osu.Game.Rulesets.Taiko/Objects/Drawables/DrawableSwell.cs

```diff
@@ -222,7 +224,7 @@ protected override void CheckForResult(bool userTriggered, double timeOffset)
                         tick.TriggerResult(false);
                 }
 
-                ApplyResult(r => r.Type = numHits > HitObject.RequiredHits / 2 ? HitResult.Ok : r.Judgement.MinResult);
+                ApplyResult(r => r.Type = numHits > HitObject.RequiredHits / 2 ? HitResult.SmallBonus : r.Judgement.MinResult);
```

Comment:
Since they're just bonus, is there anything against removing this `SmallBonus` and keep it strictly to `MaxResult`/`MinResult`? We haven't really had hitobjects give either large or small bonus.

Also we need to discuss in terms of https://github.com/ppy/osu/issues/19592, because I was hoping to include those as small/large bonuses.

Reply:
In that case what would be the metric that decides if the bonus is given? In stable points are given per tick, in lazer per swell and its properties.